OloEngine/src/OloEngine/Renderer/QualityTiering.cpp

```
@@ -0,0 +1,193 @@
+
+        // Post-process toggles
+        pp.BloomEnabled = tiering.BloomEnabled;
+        pp.BloomIterations = tiering.BloomIterations;
+        pp.FXAAEnabled = tiering.FXAAEnabled;
+        pp.DOFEnabled = tiering.DOFEnabled;
+        pp.MotionBlurEnabled = tiering.MotionBlurEnabled;
+        pp.VignetteEnabled = tiering.VignetteEnabled;
+        pp.ChromaticAberrationEnabled = tiering.ChromaticAberrationEnabled;
+    }
+
+    std::string_view QualityPresetToString(QualityPreset preset)
```

Clang (14.0.6)

[warning] use a trailing return type for this function

(modernize-use-trailing-return-type)